ci/2.7.pylintrc

```diff
@@ -1,50 +1,6 @@
 # python2.7 uses pylint 1.9.5, whose docs can be found here: http://pylint.pycqa.org/en/1.9/technical_reference/features.html#messages
 # python3.4 uses pylint 2.3.1, whose docs can be found here: http://pylint.pycqa.org/en/pylint-2.3.1/technical_reference/features.html
 
-# The source code suppresses the following error types common between python2.7 and python3.4:
```

Comment:
Thanks for removing this. Can you add a small comment on how to view the description (basically adding the command `pylint --list-msgs`)